### PR TITLE
Update pip freeze for ge.

### PR DESCRIPTION
**Pull request by coledie**
- [x] Forked repo is synced with upstream -> github shows no code delta outside of the desired.
    https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork
- [x] Tests are passing? https://github.com/microsoft/DeepGNN/blob/main/CONTRIBUTING.md#run-tests
- [x] Changelog and documentation updated.
- [x] PR is labeled using the label menu on the right side.

Previous Behavior
----------------
Pip freeze had many old versions which causes a ton of errors due to version conflicts on fresh install. I manually updated then updated freeze versions.

### CODE REVIEW

**Comment by coledie** [comment]
docs/tf/pytorch separate.

File: requirements.txt [hunk 1]
```diff
@@ -249,16 +249,17 @@ toml==0.10.2
...
 toolz==0.12.0
 torch==1.13.1
-torch-cluster==1.6.1
-torch-scatter==2.1.1
-torch-sparse==0.6.17
-torch_geometric==2.3.0
+torch-cluster==1.6.1+pt113cpu
+torch-geometric==2.3.0
+torch-scatter==2.1.1+pt113cpu
+torch-sparse==0.6.17+pt113cpu
```
Comment (coledie) [inline requirements.txt:255]: Remove + after

File: requirements.txt [hunk 2]
```diff
@@ -249,16 +249,17 @@ toml==0.10.2
...
 tornado==6.4.1
 tqdm==4.66.3
 traitlets==5.4.0
 transformers==4.38.0
 typeguard==2.13.3
-typing-extensions==4.4.0
+types-cryptography==3.3.23.2
```
Comment (coledie) [inline requirements.txt:261]: remove

File: requirements.txt [hunk 3]
```diff
@@ -33,7 +32,8 @@ azure-mgmt-resource==21.2.0
 azure-mgmt-storage==20.1.0
 azure-nspkg==3.0.2
 azure-storage-blob==12.14.0
-babel==2.10.3
+azureml-core==1.57.0
```
Comment (coledie) [inline requirements.txt:35]: remove

**Review by mortid0 — APPROVED** [review]

**Pull request merged**